content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -772,6 +773,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `HyperVContainer`: Enable
   [Hyper-V isolation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container)
   for Windows containers.
+- `IdentifyPodOS`: Allows the Pod OS field to be specified. This helps in identifying the OS of the pod
```

Comment:
What's the name of the field in a Pod spec? I would mention that detail.

Reply:
Changed the reference to `pod.spec.os.name` field directly